### ISSUE TITLE
Tooltip in performance report task stream not rendering

### ISSUE THREAD

**Issue by jrbourbeau** [dask/distributed#4515]
**What happened**:

I noticed that when we have a `performance_report` the normal hover tooltip which displays the task name and duration doesn't show up

**What you expected to happen**:

The tooltip the be present, similar to the task stream plot in the dashboard

**Minimal Complete Verifiable Example**:

```python
import dask.array as da
from distributed import Client, performance_report

if __name__ == "__main__":

    with Client() as client:
        with performance_report():
            y = da.random.normal(100).sum().compute()
```

**Anything else we need to know?**:

Locally I'm using the latest `bokeh=2.2.3` release on conda-forge

**Comment by mathdugre** [comment]
Hi @jrbourbeau, was this issue solved? Otherwise, I'd like to give it a shot.

**Comment by mathdugre** [comment]
The issue seems to be resolved with `bokeh==2.3.0`.

Those are the steps I took to verify:
1. Create file with the provided Minimal Complete Verifiable Example:
```bash
tee ./mcve.py << END
import dask.array as da
from distributed import Client, performance_report

if __name__ == "__main__":

    with Client() as client:
        with performance_report():
            y = da.random.normal(100).sum().compute()
END
```

2. Create Dockerfile:
```bash
tee ./Dockerfile << END
FROM python:3.9-slim

COPY . /src
WORKDIR /src

RUN :     && pip install -e .     && pip install "dask[complete]"     && :

# Uncomment line below to reproduce initial issue.
# RUN pip install bokeh==2.2.3


CMD pip freeze | grep bokeh; python ./mcve.py
END
```

3. Build and run the Docker container:
```bash
docker build -t distributed-4515 . && \
docker run --rm -it -v $PWD:/src distributed-4515
```

**Comment by jrbourbeau** [comment]
Ah indeed. I'm able to confirm upgrading to `bokeh=2.3.0` fixes the issue. Thanks for helping out @mathdugre!
